lib/utils/convertToDailyOverview.js

```diff
@@ -0,0 +1,52 @@
+const { createTime } = require("./convertTimestamp");
+
+const buildDailyOverview = (documents, { now }) => {
+  // delay enabled to filter events up to 6 hour
+  const delay = 21600;
```

Comment:
You can write this in uppercase.
https://javascript.info/variables#uppercase-constants
```suggestion
  const DELAY = 21600;
```

In addition, I would move this outside of this function to avoid recreating it on every call.
Or add it as parameter.

```js
const DELAY = 21600;
const buildDailyOverview = (documents, { now }) => {
```

or

```js
const buildDailyOverview = (documents, { now }, delay) => {
```

As an alternative:
`const SIX_HOURS = 21600;`

Reply:
corrected, thanks, it seems, i miss the convention this time